test/MinterGateway.t.sol

```diff
@@ -1394,18 +1399,22 @@ contract MinterGatewayTests is TestUtils {
         );
 
         uint240 activeOwedM = _minterGateway.activeOwedMOf(_minter1);
-
         uint256 maxAllowedOwedM = (collateral * _mintRatio) / ONE;
+        uint256 excessOwedM = activeOwedM - maxAllowedOwedM;
 
         uint240 penalty = uint240(
-            ((((activeOwedM - maxAllowedOwedM) * (_updateCollateralInterval - 1)) / _updateCollateralInterval) *
-                _penaltyRate) / ONE
+            (((excessOwedM * (_updateCollateralInterval - 1)) / _updateCollateralInterval) * _penaltyRate) / ONE
         );
 
         assertEq(_minterGateway.getPenaltyForMissedCollateralUpdates(_minter1), 0);
 
         vm.expectEmit();
-        emit IMinterGateway.PenaltyImposed(_minter1, _minterGateway.getPrincipalAmountRoundedDown(penalty), penalty);
+        emit IMinterGateway.UndercollateralizedPenaltyImposed(
+            _minter1,
+            uint240(excessOwedM) + 1,
+            _minterGateway.getPrincipalAmountRoundedDown(penalty),
```

Comment:
Shouldn't it be rounded up in favor of the protocol?

Reply:
I'll correct it, but it will be `_minterGateway.getPrincipalAmountRoundedUp(penalty) - 1` since contact does principal value math and tests does present value math,